tox.ini

```diff
@@ -0,0 +1,59 @@
+[tox]
+envlist =
+    check,
+    {py37},
+    test,
+    doc,
+[testenv]
+basepython =
+    py37: {env:TOXPYTHON:python3.7}
+    check: python3
+    test: python3
+    doc: python3
+passenv =
+    *
+usedevelop = true
+deps=
+    mako
+
+[testenv:check]
+deps =
+    {[testenv]deps}
+    docutils
+    twine >= 1.12.0
+    check-manifest
+    readme-renderer
+    flake8
+skip_install = true
+commands =
+    python setup.py sdist
+    twine check dist/*
+    check-manifest {toxinidir}
+    flake8 mlx tests setup.py
+
+[testenv:test]
+deps=
+    {[testenv]deps}
+    nose
+commands=
+    nosetests
```

Comment:
Since tests are run, generate coverage as well. And add a badge to readme...

Reply:
Running `coverage` 3271189f6cb582b387d844f0a41c5393ab22a53b
Added badges 65b47f60062797d1e791a22f6410c0693f7fb74d